airs-setup.sh

```diff
@@ -9,6 +9,13 @@ then
   exit 1
 fi
 
+if [[ ${#1} -gt 8 ]] || [[ ! "$1" =~ ^[a-z][a-z0-9]*$ ]] 
```

Comment:
we should enforce a min length as well

3? 4?

Reply:
I set it to 3. Thanks